lemminx-maven/src/main/java/org/eclipse/lemminx/extensions/maven/MavenLemminxExtension.java

```diff
@@ -353,10 +353,10 @@ public void stop(XMLExtensionsRegistry registry) {
 			localRepositorySearcher.stop();
 			localRepositorySearcher = null;
 		}
-		if (indexSearcher != null) {
-			indexSearcher.closeContext();
-			indexSearcher = null;
-		}
+//		if (centralSearcher != null) {
+//			centralSearcher.closeContext();
+//			centralSearcher = null;
+//		}

```

Comment:
Please remove all outdated commented code; it will greatly simplify the review,

Reply:
Done